website/repairnator-site/app/scripts/api.js

```diff
@@ -0,0 +1,13 @@
+function getApiUri(path){
+  const apiHost = 'http://51.38.235.157:4040/repairnator-mongo-api';
```

Comment:
I know I didn't do it either with the first version, but since you are refactoring it, couldn't you use [dotenv](https://www.npmjs.com/package/dotenv) to put the domain and port in an environment file used for deployment? It would be cleaner I think :)

Reply:
Good idea.
But as the website is is built by gulp how should I implement it ?
Because i won't be able to require dotenv from the webpage. Then I guess I should implement it in the gulpfile to put it in the js ?

Reply:
https://www.freshconsulting.com/how-to-organize-your-gulp-js-development-builds-for-multiple-environments/

Reply:
Thanks :)